plugins/kubernetes/app/views/kubernetes/roles/_form.html.erb

```diff
@@ -7,9 +7,15 @@
     <%= form.input :config_file %>
     <%= form.input :service_name %>
     <%= form.input :resource_name,
-          pattern: Kubernetes::RoleValidator::VALID_LABEL,
-          help: "Name for the resource this role will create (service/job), a-Z 0-9 and -"
+        pattern: Kubernetes::RoleValidator::VALID_LABEL,
+        help: "Name for the resource this role will create (service/job), a-Z 0-9 and -"
     %>
+    <% if @role.manual_deletion_required? %>
+      <div class="alert-danger">
+        <%= form.input :manual_deletion_acknowledged, as: :check_box, label: "I will manually delete the changed resource/service from all deploy groups before deploying" %>
```

Comment:
before? wouldn't that cause an outage? or will renames always cause an outage?

Reply:
in that case.. maybe mention the word "outage" / "undefined behaviour" in the check box label...